crates/ruff_python_formatter/src/expression/parentheses.rs

```diff
@@ -155,7 +155,7 @@ impl<'ast> Format<PyFormatContext<'ast>> for FormatParenthesized<'_, 'ast> {
             } else {
                 group(&format_args![
```

Comment:
Do we need this outer group anymore? There's nothing anymore that expands the parent (dangling comments always rendered a hard line break, forcing the group to expand). Or should `parenthetical_comments` render an `expand_parent`?